rc/tools/git.kak

```diff
@@ -42,6 +42,12 @@ hook -group git-show-branch-highlight global WinSetOption filetype=git-show-bran
 declare-option -hidden line-specs git_blame_flags
 declare-option -hidden line-specs git_diff_flags
 declare-option -hidden int-list git_hunk_list
+declare-option -hidden \
+    -docstring %{
+        Path of the top-level directory of the working tree currently
+        used in `git` operations from within the `kakoune`
+    } \
+    str git_work_tree_top_level
```

Comment:
why not `git_work_tree`?

Reply:
Git naming is a little confusing and at least at the time I got to the conclusion that `git_work_tree` does not mean exactly what I mean by that option. Because work tree is a work tree and a top-level of it is a top-level of it, and so on. In the end it is ok for me to have `git_work_tree` 👍🏻